interface/python_binding/CMakeLists.txt

```diff
@@ -0,0 +1,163 @@
+# Copyright 2023 Shukant Pal
+# SPDX-License-Identifier: Apache-2.0
+
+find_package (Python3 COMPONENTS Interpreter)
+file(GLOB pyktx_py_src ${CMAKE_SOURCE_DIR}/interface/python_binding/pyktx/*.py)
+list(TRANSFORM pyktx_py_src REPLACE "${CMAKE_SOURCE_DIR}/interface/python_binding/pyktx/" "${KTX_BUILD_DIR}/interface/python_binding/docs/pyktx." OUTPUT_VARIABLE pyktx_py_rst_filenames)
+list(TRANSFORM pyktx_py_rst_filenames REPLACE ".py$" ".rst" OUTPUT_VARIABLE pyktx_py_rst)
+
+message(STATUS "pyktx_py_rst is ${pyktx_py_rst}")
```

Comment:
Do we need this message? It looks like a debug message to me.

Reply:
Deleted, thanks.